src/status_im/navigation2/screens.cljs

```diff
@@ -1,6 +1,14 @@
 (ns status-im.navigation2.screens
-  (:require [status-im.navigation2.home-stack :as home-stack]))
+  (:require [status-im.ui.screens.chat.views :as chat]
+            [status-im.navigation2.home-stack :as home-stack]
+            [status-im.navigation2.stack-with-switcher :as stack-with-switcher]))
 
-(def screens [{:name      :home-stack
+;; We have to use the home screen name :chat-stack for now, for compatibility with navigation.cljs
+(def screens [{:name      :chat-stack ;; TODO(parvesh) - rename to home-stack
```

Comment:
we need to rename it in this PR right ?

Reply:
Hi @flexsurfer, we need to rename this once the `navigation2.cljs` code is ready, currently we are relying on `navigation.cljs` dispatches and it hardcodes root id name at several dispatches like [`navigation/pop-to-root-tab :chat-stack`](https://github.com/status-im/status-react/blob/2ff57ec8cdd9ebd78d674b73da90714d2f85a9bd/src/status_im/chat/models.cljs#L234). I can add a flag in those places too, but this seems cleaner for now.